pkg/controller/migmigration/pod.go

```diff
@@ -168,7 +168,7 @@ func (t *Task) ensureStagePodsCreated() (int, error) {
 	}
 
 	labelSelector := map[string]string{
-		IncludedInStageBackupLabel: t.UID(),
+		IncludedInStageBackupLabel: t.StagePodSearchLabel(),
```

Comment:
I get the method to create the label value is for consistency but thinking just t.UID() + "-pod" is sufficient and perhaps clearer.

Reply:
Cool, if you like that better I'm fine with that. That's actually how I initially had it and only moved it to remove importing `fmt` in two places. I can update this.